dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/bus/osgi/CXFExtensionBundleListener.java

```diff
@@ -0,0 +1,202 @@
+/**
```

Comment:
What did you change in this file? I suggest commenting:
// Liberty Change Start
...
// Liberty Change End

around the changes, or just
// Liberty Change

for single line changes. That's what we do in Jaxrs